### PR TITLE
Makes Scrapheap cost credits

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]: # Shit shuttle and rewards shitters and "Dank memers" to buy it.

[why]: # This is getting out of hand with a shit shuttle for the sole joke.
If people REALY want Scrapheap they should vote and shuch
Not a joke

### CODE REVIEW

**Comment by Dankseed** [comment]
Removing the only upside of the shuttle as nonantag. This won't stop people from buying if they're doing it for memes anyways. -1.

Kate Walker has purchased Standby Evacuation Vessel _"Scrapheap_ _Challenge"_ for -1000 credits. :^)

**Comment by Cw3040** [comment]
The SMART move would have been to make it emag-only

**Comment by Ziiro** [comment]
Putting the shitter shuttles behind emagging the console is highly suggested: https://github.com/OracleStation/OracleStation/pull/707

**Comment by Poojawa** [comment]
more emag-locked stuff is boring. Also hampers non-traitor antags from buying it.

**Comment by Trilbyspaceclone** [comment]
Poojawa, everyone but a few people are sick of scrapheap. Its a "Joke shuttle" that was over did.

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -227,10 +227,16 @@
 /datum/map_template/shuttle/emergency/scrapheap
 	suffix = "scrapheap"
 	name = "Standby Evacuation Vessel \"Scrapheap Challenge\""
-	credit_cost = -1000
+	credit_cost = 1000
 	description = "Due to a lack of functional emergency shuttles, we bought this second hand from a scrapyard and pressed it into service. Please do not lean to heavily on the exterior windows, they are fragile. <span class='danger'>VIRUS ALERT: This entry seems to be booby-trapped with a redirector trap. Buy this at your own risk.</span>"
 	admin_notes = "An abomination with no functional medbay, sections missing, and some very fragile windows. Surprisingly airtight."
 
+/datum/map_template/shuttle/emergency/scrapheap/prerequisites_met(mob/user)
+	if("emagged" in SSshuttle.shuttle_purchase_requirements_met)
```
Comment (Ziiro) [inline code/datums/shuttles.dm]: You only got half the code here, so this check doesn't actually do anything.

You need to add the `SSshuttle.shuttle_purchase_requirements_met |= "emagged"` to the `computer/communications.dm` in order to get this to work properly.

https://github.com/Citadel-Station-13/Citadel-Station-13/blob/d75a9cf36815aed3ed5519f1a1b3b7383213fa46/code/game/machinery/computer/communications.dm#L420-L427

Comment (Trilbyspaceclone) [reply code/datums/shuttles.dm]: Thank you for the help! I did have a changes to see that its not locked away. Thought we already had a Syda shuttle that was emag only. Guess its admin made/spawned in

**Comment by silicons** [comment]
:-1:

**Comment by Wilchenx** [comment]
>buffing emags

LoL

**Comment by Wilchenx** [comment]
also if you make it emag only then make it so there's no huge fucking message who bought the shuttle cause right now you're making it "LYNCH ME" button

**Comment by Trilbyspaceclone** [comment]
If the console is Emaged, then ANYONE can use it and buy shuttles. Maybe add in more Emag-only shuttles worth a salt and people would want someone to emag it. Also Revs(Ghost ones) can emag consoles then tell people "Pss Coms console can have scrapheap". Ninja or other antags that can emag

**Comment by Trilbyspaceclone** [comment]
To make it a non-"LYNCH ME" ware a gas mask well having your ID off will make it message "Unknown"

**Comment by RogueSteampunker** [comment]
ITT: Kate ruins everything again (^:

**Comment by Trilbyspaceclone** [comment]
@RogueSteampunker It wasn't Kate, its Was Major and Hairly that ruined the meme, Kate knew when to stop

**Comment by Poojawa** [comment]
>Poojawa, everyone but a few people are sick of scrapheap. Its a "Joke shuttle" that was over did.

So literally grudge code. ok.

**Review by deathride58 — APPROVED** [review]

**Pull request merged**